src/wallet/puzzles/did_recovery_message.clvm

```diff
@@ -0,0 +1,3 @@
+(mod (recovering_coin newpuz)
+  (r (r (c recovering_coin (c newpuz (q ())))))
```

Comment:
I can't figure out what's going on here. This seems like it will optimize to `(q ())`.